usaspending_api/disaster/management/sql/spark/disaster_covid19_file_b.py

```diff
@@ -0,0 +1,145 @@
+    financial_accounts_by_program_activity_object_class.ussgl480200_undelivered_orders_oblig_prepaid_advanced_fyb AS USSGL480200_undelivered_orders_obligations_prepaid_advanced_FYB,
+    financial_accounts_by_program_activity_object_class.ussgl488200_up_adjust_pri_undeliv_order_oblig_ppaid_adv_cpe AS USSGL488200_upward_adj_prior_year_undeliv_orders_oblig_prepaid,
+    financial_accounts_by_program_activity_object_class.gross_outlays_delivered_orders_paid_total_cpe AS gross_outlays_delivered_orders_paid_total,
+    financial_accounts_by_program_activity_object_class.gross_outlays_delivered_orders_paid_total_fyb AS gross_outlays_delivered_orders_paid_total_FYB,
+    financial_accounts_by_program_activity_object_class.ussgl490200_delivered_orders_obligations_paid_cpe AS USSGL490200_delivered_orders_obligations_paid,
+    financial_accounts_by_program_activity_object_class.ussgl490800_authority_outlayed_not_yet_disbursed_cpe AS USSGL490800_authority_outlayed_not_yet_disbursed,
+    financial_accounts_by_program_activity_object_class.ussgl490800_authority_outlayed_not_yet_disbursed_fyb AS USSGL490800_authority_outlayed_not_yet_disbursed_FYB,
+    financial_accounts_by_program_activity_object_class.ussgl498200_upward_adjust_pri_deliv_orders_oblig_paid_cpe AS USSGL498200_upward_adj_of_prior_year_deliv_orders_oblig_paid,
+    financial_accounts_by_program_activity_object_class.deobligations_recoveries_refund_pri_program_object_class_cpe AS deobligations_or_recoveries_or_refunds_from_prior_year,
+    financial_accounts_by_program_activity_object_class.ussgl487100_down_adj_pri_unpaid_undel_orders_oblig_recov_cpe AS USSGL487100_downward_adj_prior_year_unpaid_undeliv_orders_oblig,
+    financial_accounts_by_program_activity_object_class.ussgl497100_down_adj_pri_unpaid_deliv_orders_oblig_recov_cpe AS USSGL497100_downward_adj_prior_year_unpaid_deliv_orders_oblig,
+    financial_accounts_by_program_activity_object_class.ussgl487200_down_adj_pri_ppaid_undel_orders_oblig_refund_cpe AS USSGL487200_downward_adj_prior_year_prepaid_undeliv_order_oblig,
+    financial_accounts_by_program_activity_object_class.ussgl497200_down_adj_pri_paid_deliv_orders_oblig_refund_cpe AS USSGL497200_downward_adj_of_prior_year_paid_deliv_orders_oblig,
+    financial_accounts_by_program_activity_object_class.ussgl483100_undelivered_orders_oblig_transferred_unpaid_cpe AS USSGL483100_undelivered_orders_obligations_transferred_unpaid,
+    financial_accounts_by_program_activity_object_class.ussgl493100_delivered_orders_oblig_transferred_unpaid_cpe AS USSGL493100_delivered_orders_obligations_transferred_unpaid,
+    financial_accounts_by_program_activity_object_class.ussgl483200_undeliv_orders_oblig_transferred_prepaid_adv_cpe AS USSGL483200_undeliv_orders_oblig_transferred_prepaid_advanced,
+    TO_DATE((MAX(submission_attributes.published_date))) AS last_modified_date
```

Comment:
Did the output of the `TO_DATE()` function match the output from Postgres' cast to a `DATE`?

Reply:
Yes they matched, our testing has not indicated otherwise yet.